app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/fragment/NavigationFragment.java

```diff
@@ -122,6 +135,23 @@ public void onNavigationRunning() {
     // no-op
   }
 
+  @Override
+  public void onProgressChange(Location location, RouteProgress routeProgress) {
+    boolean isInTunnel = routeProgress.inTunnel();
+    boolean wasInTunnel = wasInTunnel();
```

Comment:
I like the variable naming here - if possible, let's us the same naming in `RerouteActivity`